nameko_grpc/streams.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, stream_id):
     def exhausted(self):
         """A stream is exhausted if it is closed and there are no more messages to be
         consumed or bytes to be read.
+
+        When a stream is closed we append the STREAM_END item or a GrpcError, so an
+        exhausted stream will possibly still have 1 item left in the queue, so we
+        must check for that.
```

Comment:
This doesn't seem to be reflected in the implementation?

Reply:
You're correct, the comment is not needed.